### ISSUE TITLE
emojis broken

### ISSUE THREAD

**Issue by Astrogem2** [tgstation/tgstation#80581]
Reporting client version: 515.1621

## Issue Summary
Emojis not appearing in ooc

## Round ID:
[221140](https://scrubby.melonmesa.com/round/221140)

## Testmerges:
- [Save json savefiles to disk in pretty print form](https://www.github.com/tgstation/tgstation/pull/80512)
- [(Testmerge Ready) New Antagonist: Tiger Cooperative Fanatic](https://www.github.com/tgstation/tgstation/pull/80313)
- [[TM ONLY] Remove 'natural' beheading](https://www.github.com/tgstation/tgstation/pull/80439)

## Reproduction:

Basically just typing like :pmoth: in ooc, and instead of putting down the moth plush, it just puts down ":pmoth:"
![image](https://github.com/tgstation/tgstation/assets/101240878/4a2a4e23-8a1d-4689-a6be-71160138138b)

**Comment by Mothblocks** [comment]
I think related to recent changes to cache all the spritesheets, including emoji.

These lines in particular stand out to me:

https://github.com/tgstation/tgstation/blob/fbb125fd0871c15f84bf377de2c5536439b6d03c/code/modules/emoji/emoji_parse.dm#L20-L28

**Comment by Mothblocks** [comment]
![image](https://github.com/tgstation/tgstation/assets/35135081/f62390a4-3410-43a4-a29f-35eeace4563d)

**Comment by Mothblocks** [comment]
VV on GLOB.asset_datums[/datum/asset/spritesheet/chat]:

![image](https://github.com/tgstation/tgstation/assets/35135081/df1db626-88ec-4866-9f35-9bf23ee36ead)

**Comment by Mothblocks** [comment]
Likely also breaking commendation hearts
![image](https://github.com/tgstation/tgstation/assets/35135081/39c49aeb-ac83-4815-b37c-5dc83e45ad01)
